src/list.js

```diff
@@ -79,7 +58,33 @@ export default class List {
     localStorage.setItem('todo-list', JSON.stringify(this.items));
   }
 
+  addItem(name) {
+    this.items.push({
+      description: name,
+      completed: false,
+      index: this.items.length,
+    });
+    this.displayItems();
+  }
+
+  editItem(task, index) {
+    this.items[index - 1].description = task;
+    this.saveItems();
```

Comment:
[OPTIONAL]
Although these functions work just great 👏🏽 , you might find it handy to know that within the `this.saveItems` function:
- you can also sort the tasks in ascending order of indexes this way `this.items.sort((a, b) => a.index - b.index);` This is because arrays of objects can be sorted by comparing the value of one of their properties.